servers/xr_server.cpp

```diff
@@ -352,6 +362,53 @@ PackedStringArray XRServer::get_suggested_pose_names(const StringName &p_tracker
 	return arr;
 }
 
+void XRServer::add_face_tracking_provider(const StringName &p_path, Ref<XRFaceTrackingProvider> p_face_tracking_provider) {
```

Comment:
As you point out, it is just a name and we're calling it a path because we're thinking of putting path-like names such as "/user/head" or "/remote/netid_2795269437". I chose to call it path to match the proposal, but I have no preference in this regard unless we were to extend the API with some sort of path-like feature such as a `find("/remote/*") -> Array[XRFaceTrackingProvider]`, but even that can be considered as being name-matching with the `String.match` like rules rather than being true path-traversal.

Reply:
I'd personally prefer "name" rather than "path". With trackers we call them "name" on `XRServer` (they are pathes when using OpenXR, but won't necessarily be for other XR interfaces). Being consistent with how we did trackers makes sense to me.

Reply:
I switched this over to "tracker_name" everywhere - to match the naming convention used for the other types of trackers.